src/amo/components/AddonReviewList/index.js

```diff
@@ -95,9 +103,14 @@ export class AddonReviewListBase extends React.Component<Props> {
     if (!reviews || locationChanged) {
       dispatch(
         fetchReviews({
-          addonSlug: params.addonSlug,
+          addonSlug: match.params.addonSlug,
           errorHandlerId: errorHandler.id,
-          page: location.query.page,
+          // TODO: so, there is a test case (`it dispatches fetchReviews with
+          // an invalid page variable`) that conflicts with `fetchReviews()`
+          // requiring a page of type `number`. We should decide whether `page`
+          // can be anything OR restrict to integer values.
+          // $FLOW_FIXME
```

Comment:
I think you could just change this action to accept a string `page`. The [API page param](https://github.com/mozilla/addons-frontend/blob/master/src/amo/api/reviews.js#L143) may need to change to a string as well but that's fine.